templates/definition/meter/eastron-sdm230.yaml

```diff
@@ -0,0 +1,22 @@
+template: eastron-sdm230
```

Comment:
Warum brauchts das 230 wenn wir bisher beide Zähler in einem Tempalte abdecken konnten?

Reply:
Bitte keine Luftschlösser.

Welche anderen für evcc _relevanten_ Parameter unterscheiden sich?

Phasenwinkel & Co. sind völlig irrelevant, schon andere Dinge abseits Power & Energy (Import) sind für Nicht-3p-Zähler schon nicht relevant.

Reply:
Du hast bestimmt recht. Wir versuchen trotzdem, Differenzierungen erst zu machen wenn wir sie brauchen. Muss ja alles gewartet werden…